src/data/finset/sigma.lean

```diff
@@ -52,30 +52,30 @@ section sigma_lift
 variables [decidable_eq ι]
 
 /-- Lifts maps `α i → β i → finset (γ i)` to a map `Σ i, α i → Σ i, β i → finset (Σ i, γ i)`. -/
-def sigma_lift₂ (f : Π ⦃i⦄, α i → β i → finset (γ i)) (a : sigma α) (b : sigma β) :
```

Comment:
I'd argue for keeping the 2 in the name

Reply:
Even without a unary version in sight? That sounds like unneeded and confusing pain. Kind of feels like renaming `eq` to `eq₂` because it is binary.

Reply:
Presumably the unary `sigma_lift` would look like `(∀ i, α i → finset (γ i)) → Σ i, α i → finset (Σ i, γ i)`. I don't think that one already exists, right?

Reply:
As noted by Bhavik, the unary version is a bit lame. If we generalize it to all alternative monads, then yeah sure we can rename that to `sigma.lift₂` and have the unary version called `sigma.lift`.

Reply:
Ok, I guess if we're planning to rename it anyway then this discussion is a bit pointless.